compiler/gfortran.cmake

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2017, NeroBurner
+# All rights reserved.
+
+if(DEFINED POLLY_COMPILER_Fortran_CMAKE)
+  return()
+else()
+  set(POLLY_COMPILER_Fortran_CMAKE 1)
+endif()
+
+find_program(CMAKE_Fortran_COMPILER gfortran)
+
+if(NOT CMAKE_Fortran_COMPILER)
+  polly_fatal_error("gfortran not found")
```

Comment:
This toolchain can be used for projects without Fortran requirement so we should not crash here.